packages/widget/src/components/transfer/fungible/transfer-detail/transfer-detail.ts

```diff
@@ -49,19 +56,46 @@ export class FungibleTransferDetail extends BaseComponent {
     }
   }
 
-  getFee(): string {
+  getGasFeeParams(): {
+    decimals?: number;
+    symbol: string;
+  } {
+    let decimals = undefined;
+    let symbol = '';
+    if (this.sourceDomainConfig) {
+      decimals = Number(this.sourceDomainConfig.nativeTokenDecimals);
+      symbol = this.sourceDomainConfig.nativeTokenSymbol.toUpperCase();
+    }
+    return { decimals, symbol };
+  }
+
+  getSygmaFee(): string {
     if (!this.fee) return '';
-    const { symbol, decimals } = this.getFeeParams(this.fee.type);
+    const { symbol, decimals } = this.getSygmaFeeParams(this.fee.type);
     const { fee } = this.fee;
     let _fee = '';
 
     if (decimals) {
-      _fee = tokenBalanceToNumber(fee, decimals, 4);
+      _fee = tokenBalanceToNumber(BigNumber.from(fee.toString()), decimals, 4);
     }
 
     return `${_fee} ${symbol}`;
   }
 
+  getEstimatedGasFee(): string {
+    if (!this.estimatedGasFee) return '';
+    const { symbol, decimals } = this.getGasFeeParams();
+
+    let gasFee;
```

Comment:
I would expect here the linter to suggest to use const, since this variable is not being used outside the block. I think you can just declare `const gasFee` in the block, and then return it.